schema/sale/index.js

```diff
@@ -150,6 +150,41 @@ const SaleType = new GraphQLObjectType({
       description: {
         type: GraphQLString,
       },
+      display_timely_at: {
+        type: GraphQLString,
+        resolve: ({ is_live_open, live_start_at, start_at, end_at }) => {
+          if (end_at < moment()) {
```

Comment:
maybe micro-optimization: is it worth taking `moment()` out and making it `const now = moment()` the functions is used a bunch